### ISSUE TITLE
Buttons are too small on tablets

### ISSUE THREAD

**Issue by JoshuaRamey** [lrorpilla/jidoujisho#344]
I'm using jidoujisho to read manga on a tablet and it's great but the buttons in the dictionary window are small so it takes a million attempts to open the card creation window.

Some buttons elsewhere are small as well, like for changing sources in the reader tab or media tab.

Would it be possible to update button sizes for larger displays?

**Comment by ONEsuzuki** [comment]
Yeah I agree just started using the app on my tablet, dictionary buttons are especially small.

**Comment by d-najd** [comment]
Don't agree, I tried the app on my tablet and it doesn't look like they are too small, care to point out which buttons you deem are too small?

**Comment by ONEsuzuki** [comment]
@d-najd The quick add buttons I feel are too small, I have an Xiaomi Pad 5 tablet and I either hit the wrong button or have to keep taping the quick-export button because it's too small and fiddly to press.

**Comment by Yan3ku** [comment]
https://github.com/arianneorpilla/jidoujisho/issues/402#issue-2526012337
I some issue with button and this may be UI related.
